#include "exception.hpp"

exception::exception(const char* desc)
	: desc(desc)
{}